fenwick_tree/cpp/fenwick_tree.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+#define SIZE 100000
+
+class FenwickTree {
+private:
+	vector<int> v;
+	int len;
+
```

Comment:
Constructors callable with one argument should be marked explicit.  [runtime/explicit] [5]

*Origin: CPPLintBear, Section: `all.cpplint`.*